.github/workflows/praul.yaml

```diff
@@ -0,0 +1,22 @@
+name: PRaul
```

Comment:
HAaha, I love the name 

@luarx, it's now a thing !

Reply:
Hahaha thank you guys for this recognition! 
It's a pleasure :D